src/grandorgue/CMakeLists.txt

```diff
@@ -122,7 +122,6 @@ gui/GOGUIPanel.cpp
 gui/GOGUIPanelWidget.cpp
 gui/GOGUIRecorderPanel.cpp
 gui/GOGUISequencerPanel.cpp
-gui/GOGUISizeKeeper.cpp
```

Comment:
I just wondering why moving from gui to size (as this could be ambiguous about what the size is for)

Reply:
Because this PR adds more files related with sizes. I haven't found any other short word for describing what size is for. GUI is not more accepted after moving.